<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arrange_games</class>
 <widget class="QWidget" name="arrange_games">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="team1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="team2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="لطفا تاریخ و ساعت مسابقه ی خود را به میلادی وارد نمایید"/>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>1752</year>
     <month>9</month>
     <day>14</day>
    </date>
   </property>
   <property name="minimumTime">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::AmPmSection</enum>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="our_team_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sakkal Majalla</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>بازی های ما</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>اضافه کردن بازی</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="our_games">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>601</width>
     <height>192</height>
    </rect>
   </property>
   <property name="toolTip">
    <string> برای تغییر چیزی تغییرات را در جدول اعمال کنید .</string>
   </property>
   <column>
    <property name="text">
     <string>تیم اول</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>نتیجه</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تیم دوم</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تاریخ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ساعت</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>حذف</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="tour_naem_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Urdu Typesetting</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="tour_name">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>نام تورنومنت را انتخاب نمایید.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tournoments">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
